Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -594,48 +594,48 @@ Don't have a 4chan Pass?<br>
     <string name="thread_layout_remove_whole_chain">Remove whole chain</string>
     <string name="thread_layout_hide_post">Hide post</string>
     <string name="thread_layout_remove_post">Remove post</string>
-    <string name="thread_layout_hide_whole_chain_as_well">Would you like to hide the whole reply chain as well?</string>
-    <string name="thread_layout_remove_whole_chain_as_well">Would you like to remove the whole reply chain as well?</string>
+    <string name="thread_layout_hide_whole_chain_as_well">Hide the whole reply chain too?</string>
+    <string name="thread_layout_remove_whole_chain_as_well">Remove the whole reply chain too?</string>
     <string name="view_removed_posts">View removed posts</string>
     <string name="restore_posts">Restore posts</string>
     <string name="restored_n_posts">Restored %1$d posts</string>
     <string name="no_removed_posts_for_current_thread">No removed posts found for the current thread</string>
 
     <!-- Captcha settings -->
     <string name="captcha_layout_v2_verify_button_text">Verify</string>
-    <string name="captcha_layout_v2_use_old_captcha">Use old\ncaptcha</string>
+    <string name="captcha_layout_v2_use_old_captcha">Use old\nCAPTCHA</string>
 
-    <string name="captcha_layout_v2_you_have_to_select_at_least_one_image">You have to select at least one image (if you see a captcha that does not have any matching images, then it is probably a bug; fill an issue and use the old captcha)</string>
-    <string name="captcha_layout_v2_verification_already_in_progress">Verification is already in progress</string>
-    <string name="captcha_layout_v2_captcha_request_is_already_in_progress">Captcha request is already in progress</string>
-    <string name="captcha_layout_v2_you_are_requesting_captcha_too_fast">You are requesting captcha too fast</string>
+    <string name="captcha_layout_v2_you_have_to_select_at_least_one_image">Select at least one image (if you see a CAPTCHA that does not have any matching images you can file a bug and use the old one)</string>
+    <string name="captcha_layout_v2_verification_already_in_progress">Verification already in progress…</string>
+    <string name="captcha_layout_v2_captcha_request_is_already_in_progress">CAPTCHA request already in progress…</string>
+    <string name="captcha_layout_v2_you_are_requesting_captcha_too_fast">You requested too many CAPTCHAs in a short timespan.</string>
     <string name="image_url_copied_to_clipboard">Image URL copied to clipboard</string>
     <string name="delete_site_dialog_title">Delete site?</string>
-    <string name="delete_site_dialog_message">Are you sure you want to delete "%1$s"? This action will delete the site and everything that is associated with it. The app will be restarted. Do you wish to proceed?</string>
+    <string name="delete_site_dialog_message">Really delete "%1$s" (meaning the site and everything associated with it and then restart the app)?</string>
```

Comment:
The original message could be too long, but I did like the "do you want to proceed?" part to affirm action.

Reply:
Will try to revise